README.md

```diff
@@ -19,8 +19,8 @@
   * remove redundant code (e.g. `public` keyword)
   * remove unused imports
   * skip default ImageVector parameters
-  * support generation as [Backing field or with Lazy property](https://github.com/ComposeGears/Valkyrie/tree/task/update-readme?tab=readme-ov-file#export-formats)
```

Comment:
This is broken, and we should avoid using absolute refs here.

Reply:
Maybe, I did it in IntelliJ IDEA and everything worked 🙂
Thanks for this!